packages/build/src/build-and-upload.spec.ts

```diff
@@ -99,15 +116,41 @@ describe('buildAndRelease', () => {
     );
   });
 
-  it('releases to downloads centre if a public release', async() => {
+  it('releases to barque if a public release', async () => {
+    githubRepo = createStubRepo({
+      shouldDoPublicRelease: sinon.stub().returns(Promise.resolve(true))
+    });
+
+    barque = createStubBarque({
+      releaseToBarque: sinon.stub().returns(Promise.resolve(true))
+    });
+
+    await buildAndUpload(
+      config,
+      githubRepo,
+      barque,
+      compileAndZipExecutable,
+      uploadToEvergreen,
+      uploadToDownloadCenter
+    );
+
+    expect(barque.releaseToBarque).to.have.been.called;
+  });
+
+  it('releases to downloads centre if a public release', async () => {
     githubRepo = createStubRepo({
       shouldDoPublicRelease: sinon.stub().returns(Promise.resolve(true)),
       releaseToGithub: sinon.stub().returns(Promise.resolve(true))
     } as any);
 
+    barque = createStubBarque({
+      releaseToBarque: sinon.stub().returns(Promise.resolve(true))
```

Comment:
I think you can do this if you like:

```suggestion
      releaseToBarque: sinon.stub().resolves(true)
```